torch/_inductor/triton_bundler.py

```diff
@@ -159,8 +160,10 @@ def collect(
                             counters["inductor"]["triton_bundler_save_kernel"] += 1
                         except Exception:
                             log.debug("failed to collect triton kernel", exc_info=True)
-                        if filename.endswith(".cubin"):
-                            # Each kernel has bunch of files like .cubin, .json, .ttir
+                        file_extension = os.path.splitext(filename)[1]
+                        if file_extension in GPU_KERNEL_BIN_EXTS.values():
```

Comment:
GPU_KERNEL_BIN_EXTS misses a period resulting in this branch never matching.

Reply:
@etaf , why could the logic pass the test cases?